consensus/polybft/checkpoint_manager.go

```diff
@@ -0,0 +1,177 @@
+package polybft
+
+import (
+	"fmt"
+	"math/big"
+	"strconv"
+
+	"github.com/0xPolygon/polygon-edge/command/rootchain/helper"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/abi"
+)
+
+var (
+	// currentCheckpointIDMethod is an ABI method object representation for
+	// currentCheckpointId getter function on CheckpointManager contract
+	currentCheckpointIDMethod, _ = abi.NewMethod("function currentCheckpointId() returns (uint256)")
+
+	// submitCheckpointMethod is an ABI method object representation for
+	// submit checkpoint function on CheckpointManager contract
+	submitCheckpointMethod, _ = abi.NewMethod("function submitCheckpoint(" +
+		"uint256 chainID, bytes aggregatedSignature, bytes validatorsBitmap, " +
+		"uint256 epochNumber, uint256 blockNumber, bytes32 blockHash, uint256 blockRound," +
+		"bytes32 eventRoot, tuple(address _address, uint256[4] blsKey)[] nextValidators" + ")")
+
+	burazBrat, _ = abi.NewMethod(
```

Comment:
Remove this :joy:

Reply:
Fixed [6134a1a](https://github.com/0xPolygon/polygon-edge/pull/861/commits/6134a1a9e438cb2c9a4174f781cdb85b0a2bf02c)